src/controllers/local-storage-page-controller.js

```diff
@@ -46,34 +82,80 @@ angular.module('ToDo').controller('localStoragePageController',
                     selectedLanguage() {
                         return $scope.selectedLanguage;
                     },
-                    todosMemoryValue() {
-                        return ctrlConnect.getTodos();
+                    memoryValue() {
+                        const arr = [];
+                        arr[0] = ctrlConnect.getTodos();
+                        arr[1] = ctrlConnect.getComplTodos();
+                        return arr;
+                    },
+                    obj() {
+                        return $scope.userChoise;
                     },
                 },
             });
             modalInstance.result.then(function (answer) {
                 let res = [];
                 switch (answer) {
                     case MODAL_ANSWERS.SAVE:
-                        res = JSON.parse(localStorage.getItem('todos'));
-                        $scope.setInService(res);
+                        res = JSON.parse(localStorage.getItem('allTodos'));
+                        $scope.setInService($scope.userChoise, res);
                         alertBox.addAlert($scope.translation.LOAD_FROM_STOR);
                         break;
                     case MODAL_ANSWERS.MERGE:
-                        if (localStorage.getItem('todos') !== null) {
-                            res = ctrlConnect.getTodos().concat(JSON.parse(localStorage.getItem('todos')));
-                            $scope.setInService(res);
+                        if (localStorage.getItem('allTodos') !== null) {
+                            res[0] = JSON.parse(localStorage.getItem('allTodos'))[0];
```

Comment:
load and parse json file twice not optimize decision. You can use destructuring for that.